extensions/amp-form/0.1/test/test-utils.js

```diff
@@ -53,6 +54,7 @@ describes.realWin('amp-form-utils', {}, env => {
       });
 
       it("returns true if textarea's value matches its default value", () => {
+        textarea.value = 'default';
```

Comment:
It's okay to use a hardcoded string here as it's pretty much self-explained, but why not just use textarea.defaultValue? Would that work?

Reply:
I deliberately chose to use a string value here, as I want the default value to not be empty. I could extract that into a named constant if that works better for you?

Reply:
Okay I see, that should be fine.
These tests are pretty simple so I won't push further. But technically, these tests are still relying on the "hidden" fact that the default value is string 'default'. When you are working on some messier unit tests, you might consider moving those logic from `beforeEach` to tests themselves.

When writing unit tests, then most important thing is whether the test is clear to readers (i.e. if there are hidden facts) and then the cleanest/simplest of the tests.